tests/integration/targets/ios_net_put_get/tasks/main.yaml

```
@@ -0,0 +1,2 @@
+---
+- { include: cli.yaml, tags: ['cli'] }
```

```suggestion
- { include: cli.yaml, tags: ['network_cli'] }
```

Though I generally prefer

```suggestion
- include: cli.yaml
  tags:
    - network_cli
```